liquidator/liquidator.js

```diff
@@ -2,7 +2,15 @@
 // wallet to run the liquidations. Future versions will deal with generating additional synthetic tokens from EMPs as the bot needs.
 
 class Liquidator {
-  constructor(logger, expiringMultiPartyClient, gasEstimator, account) {
+  /**
+   * @notice Constructs new Liquidator bot.
+   * @param {*Object} logger Module used to send logs.
+   * @param {*Object} expiringMultiPartyClient Module used to query EMP information on-chain.
+   * @param {*Object} gasEstimator Module used to estimate optimal gas price with which to send txns.
+   * @param {*String} account Ethereum account from which to send txns.
+   * @param {?Object} config Optional. Contains fields with which constructor will attempt to override defaults.
```

Comment:
what's the canonical way to signal that a parameter is optional?

Reply:
https://stackoverflow.com/a/29441515 ? not sure if we're following JSDoc, though. Where did the `*`s come from?

Reply:
The `*`'s were auto inserted from VS Code so I thought they were canonical.

Reply:
The `*`s are cool with me -- I don't know anything about js docs. Assuming they come from documentation generator specification. LGTM